examples/counter_router/src/main.rs

```diff
@@ -0,0 +1,109 @@
+#[macro_use]
+extern crate stdweb;
+#[macro_use]
+extern crate yew;
+
+use yew::prelude::*;
+use yew::services::console::ConsoleService;
+use yew::services::router::{RouterService, RouteInfo};
+
+use stdweb::web::Date;
+
+struct Context {
+    console: ConsoleService,
+    router: RouterService<Context, Model>,
```

Comment:
I really like the idea of specifying what component the router will be bound to.

Reply:
Ya, it was required because I held onto the raw callback function. Unfortunately I don't think that lead to a very clean API so is no longer necessary.